content/developer/howtos/upgrade_custom_db.rst

```diff
@@ -0,0 +1,262 @@
+:show-content:
+
+.. _upgrade_custom/upgrade_custom_db:
+
+===============================
+Upgrading a customized database
+===============================
+
+Upgrading to a new version of Odoo can be challenging, especially if the database you work on
+contains custom modules. This page intent is to explain the technical process of upgrading a
+database with customized modules. For a standard overview, please refer to the
```

Comment:
What is a "standard overview" in this context?

Maybe: "For the functional process/perspective"?